### PR TITLE
Make torrent file names on the table selectable

### LINKED ISSUE #8148: torrent file names should be selectable
**Describe the issue you encountered:** torrent file names on sortableTable should be selectable.

- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10 32 bit

- Brave Version (revision SHA): 0.14.1 Preview 1

- Steps to reproduce:
    1. Open https://webtorrent.io/free-torrents/
    2. Click any torrent
    3. Start downloading
    
- Actual result: The title and the file names on the table cannot be selected

- Expected result: they should be selectable

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

### PR DESCRIPTION
**Pull request by luixxiul**
Closes #8148

Also:
- Added testIds to the buttons for future work
- Made missingFilesList, loadingFilesList, torrentStats, and legalNotice unselectable

Auditors:

Test Plan:
1. Open https://webtorrent.io/free-torrents
2. Click "Big Buck Bunny (magnet link)"
3. Make sure nothing is selectable
4. Click "Start download"
5. Make sure only the file names on the table are selectable

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

### CODE REVIEW

File: js/webtorrent/components/torrentFileList.js [hunk 1]
```diff
@@ -54,9 +56,20 @@ class TorrentFileList extends React.Component {
...
         : '&ix=' + ix
       const href = torrentId + suffix
-      return <a href={href}>{file.name}</a>
+      return <a className={css(styles.userSelect)} href={href}>{file.name}</a>
     }
   }
 }
 
+const styles = StyleSheet.create({
+  userSelect: {
```
Comment (NejcZdovc) [inline js/webtorrent/components/torrentFileList.js]: Please move this two definition into the common file and use it in all files

Comment (luixxiul) [inline js/webtorrent/components/torrentFileList.js]: Updated. Will squash later.

Comment (NejcZdovc) [inline js/webtorrent/components/torrentFileList.js]: Thank you

**Comment by luixxiul** [comment]
@NejcZdovc done

**Comment by srirambv** [comment]
Save file icon is also selectable. image

**Pull request merged**